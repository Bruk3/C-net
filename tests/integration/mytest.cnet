int main()
{
    {
        int x = 5;
        int y = 15;
        {
            {
                int y = 20;

                x+=1;
                stdout.writeln("x is " + soi(x));
                y+=1;
                stdout.writeln("y is " + soi(y)); 
                int z = x + y; 
                stdout.writeln("z(27): " + soi(z)); 
            }            
            int z = x + y; // outer block x and y
            stdout.writeln("x is " + soi(x));
            stdout.writeln("y is " + soi(y));
            stdout.writeln("z(22): " + soi(z)); 
        }
    }
    return 0;
}