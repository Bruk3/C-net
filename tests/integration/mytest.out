x is 6
y is 21
z(27): 27
x is 6
y is 21
z(16): 27
